BDArmory/Weapons/Missiles/MissileLauncher.cs

```diff
@@ -3572,52 +3612,57 @@ public override string GetInfo()
                 {
                     case "MultiMissileLauncher":
                         {
-                            warheadType = WarheadTypes.Launcher;
+                            warheadType = WarheadTypes.Launcher; //Why is this getting set here? warHeadType is already set in onStart()
```

Comment:
Because this is in `GetInfo`, which is called during KSP startup when parsing the missile configs, whereas the one in `OnStart` is only called when the part is instantiated in-flight or in the editor and this allows for detecting legacy missiles via the `warheadType == WarheadTypes.Kinetic` check below (line 3671).